lib/CustomPlugin.ts

```diff
@@ -0,0 +1,24 @@
+import { Router } from 'express'
+import MqttClient from './MqttClient'
+import { ModuleLogger } from './logger'
+import ZwaveClient from './ZwaveClient'
+
+export type PluginContext = {
```

Comment:
This should be an interface, not a type

Reply:
In some cases like this I don't get why I should use interface instead of a type, mine it's a newbe question but they seems to be quite the same. The only difference is that interface can be extended but the type can use `&` operator